app/Dockerfile

```diff
@@ -26,6 +27,11 @@ RUN if [ "$ASPACE_VERSION" = "latest" ]; then \
     unzip -q /archivesspace-$ARCHIVESSPACE_VERSION.zip -d / && \
     mv /archivesspace/config/config.rb /archivesspace/config/config-$ARCHIVESSPACE_VERSION.rb
 
+RUN xmlstarlet edit -L \
+    --subnode "//config" -t elem -n indexConfig \
+    --subnode "//config/indexConfig" -t elem -n lockType -v "\${solr.lock.type:none}" \
+    /archivesspace/solr/solrconfig.xml
```

Comment:
Looks like either I have to add this to `app/Dockerfile` also OR set `AppConfig[:solr_verify_checksums]` to `false` in `config.rb`. I guess either is fine? But seems like they added the checksum guard for a reason...

Reply:
the checksum is probably there to prevent malware from messing with the config.rb file

Reply:
Makes sense.